docs/docs/how-tos/get-started/4-human-in-the-loop.md

```diff
@@ -0,0 +1,265 @@
+# Add human oversight to the chatbot    
```

Comment:
maybe just `human-in-the-loop`? or `human review`? i don't think oversight is a common term for this

Reply:
Changed to "Add HITL controls to the chatbot" but also asked the team